tests/Unit/ProtoTests/ClientTest.php

```diff
@@ -35,4 +36,12 @@ public function testUnsupportedTransportThrowsException()
             'credentials' => new InsecureCredentialsWrapper(),
         ]);
     }
+
+    public function testGrpcOnlyClientThrowsExceptionForRestTransport()
+    {
+        $this->expectException(ValidationException::class);
+        $this->expectExceptionMessage('Unexpected transport option "rest". Supported transports: grpc, grpc-fallback');
+
+        $client = new BasicGrpcOnlyClient(['transport' => 'rest']);
```

Comment:
I think we can fix the failing tests by passing `Google\ApiCore\InsecureCredentialsWrapper`:

```suggestion
        $client = new BasicGrpcOnlyClient([
            'transport' => 'rest',
            'credentials' => new InsecureCredentialsWrapper(),
        ]);
```